### DISCUSSION TITLE
Issues updating from 0.0.2

### DISCUSSION THREAD

**Discussion by snasser117** [Comfy-Org/ComfyUI#4467] [Q&A]
I am having issues updating my stand-alone ComfyUI install. As you can see below, I'm getting multiple errors regarding modules not found and some notes that did not import correctly. Around the batch file that only updates the install (update_comfyui.bat) and not the Python dependencies. I was concerned that it might cause issues with my invoke AI install but that was pure speculation on my part. Should I just run the full upgrade including Python? Thank you!

E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
Total VRAM 12288 MB, total RAM 130839 MB
pytorch version: 2.0.1+cu118
xformers version: 0.0.20
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 : cudaMallocAsync
Using xformers cross attention
Updated the windows standalone package updater.
****** User settings have been changed to be stored on the server instead of browser storage. ******
****** For multi-user setups add the --multi-user CLI argument to enable multiple user profiles. ******
[Prompt Server] web root: E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\web
Adding extra search path checkpoints C:\Users\stable-diffusion-webui\models/Stable-diffusion
Adding extra search path configs C:\Users\stable-diffusion-webui\models/Stable-diffusion
Adding extra search path vae C:\Users\stable-diffusion-webui\models/VAE
Adding extra search path loras C:\Users\stable-diffusion-webui\models/Lora
Adding extra search path loras C:\Users\stable-diffusion-webui\models/LyCORIS
Adding extra search path upscale_models C:\Users\stable-diffusion-webui\models/ESRGAN
Adding extra search path upscale_models C:\Users\stable-diffusion-webui\models/RealESRGAN
Adding extra search path upscale_models C:\Users\stable-diffusion-webui\models/SwinIR
Adding extra search path embeddings C:\Users\stable-diffusion-webui\embeddings
Adding extra search path hypernetworks C:\Users\stable-diffusion-webui\models/hypernetworks
Adding extra search path controlnet C:\Users\stable-diffusion-webui\models/ControlNet
Traceback (most recent call last):
  File "E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1993, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy_extras\nodes_upscale_model.py", line 3, in <module>
    from spandrel import ModelLoader, ImageModelDescriptor
ModuleNotFoundError: No module named 'spandrel'

Cannot import E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy_extras\nodes_upscale_model.py module for custom nodes: No module named 'spandrel'
Traceback (most recent call last):
  File "E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1993, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy_extras\nodes_canny.py", line 1, in <module>
    from kornia.filters import canny
ModuleNotFoundError: No module named 'kornia'

Cannot import E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy_extras\nodes_canny.py module for custom nodes: No module named 'kornia'
Traceback (most recent call last):
  File "E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1993, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy_extras\nodes_morphology.py", line 4, in <module>
    from kornia.morphology import dilation, erosion, opening, closing, gradient, top_hat, bottom_hat
ModuleNotFoundError: No module named 'kornia'

Cannot import E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\comfy_extras\nodes_morphology.py module for custom nodes: No module named 'kornia'
E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\python_embeded\lib\site-packages\torchaudio\backend\utils.py:74: UserWarning: No audio backend is available.
  warnings.warn("No audio backend is available.")

Import times for custom nodes:
   0.0 seconds: E:\ComfyUI_windows_portable_nvidia_cu118_or_cpu\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py

WARNING: some comfy_extras/ nodes did not import correctly. This may be because they are missing some dependencies.

IMPORT FAILED: nodes_upscale_model.py
IMPORT FAILED: nodes_canny.py
IMPORT FAILED: nodes_morphology.py

This issue might be caused by new missing dependencies added the last time you updated ComfyUI.
Please run the update script: update/update_comfyui.bat

Starting server

To see the GUI go to: http://127.0.0.1:8188

**Comment by ltdrdata** [comment]
You have to install `ComfyUI/requirements.txt`.
```
.\python_embeded\python.exe -m pip install -r ComfyUI/requirements.txt
```
